### ISSUE TITLE
restart_then_repair_node nemesis: db errors caused by RAID setup when scylla is running

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla-cluster-tests#1298]
### Prerequisites

* [x] Are you rebased to master ?
* [x] Is it reproducible ?
* [x] Did you perform a cursory search if this issue isn't opened ?

### Versions
* **SCT**: branch-3.1
* **scylla**: scylla 3.1.0.rc6

### Logs
* Test ID: 0d3e8b0f-da18-4681-bac7-00bda43616e8
* job log: https://cloudius-jenkins-test.s3.amazonaws.com/0d3e8b0f-da18-4681-bac7-00bda43616e8/job_log.zip
* events log: https://cloudius-jenkins-test.s3.amazonaws.com/0d3e8b0f-da18-4681-bac7-00bda43616e8/events_log.zip

### Description

```
09-09 13:40:12,743 >>>Started random_disrupt_method restart_then_repair_node


13:41:45,455 f:remote.py       l:213  c:sdcm.remote          p:DEBUG > RemoteCmdRunner [centos@172.30.0.110]: Running command "sudo /usr/lib/scylla/scylla-ami/scylla_create_devices"...
13:41:48,701 f:remote.py       l:213  c:sdcm.remote          p:DEBUG > RemoteCmdRunner [centos@172.30.0.110]: Running command "sudo systemctl stop scylla-server.service"...

13:43:14+00:00 !ERR     | scylla: [shard 11] compaction_manager - compaction failed: std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: mkdir failed: No such file or directory [/var/lib/scylla/data/system_schema/tables-afddfb9dbc1e30688056eed6c302ba09/0000000000000361.sstable]): retrying

13:43:17+00:00 [shard 4] table - failed to write sstable /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/mc-256-big-Data.db: std::filesystem::__cxx11::filesystem_error (error system:2, filesystem error: mkdir failed: No such file or directory [/var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/0000000000000256.sstable])

13:43:27,750 f:cluster.py      l:1216 c:sdcm.cluster         p:DEBUG > 2019-09-09T13:43:19+00:00  ip-172-30-0-110 systemctl start scylla-server.service
13:43:21,088 f:nemesis.py      l:211  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: Waiting scylla services to start after node restart

09-09 13:46:54,024 <<<Finished random_disrupt_method restart_then_repair_node
```

### Steps to Reproduce

1. run longevity test, it occurs in restart_then_repair_node nemesis

**Expected behavior:** The error can be ignored, we don't want them fail the test.
**Actual behavior:** The errors are collected, and it would fail the test.

/CC @fruch

**Comment by fruch** [comment]
I think @bentsi is fixing this in #1297

**Comment by amoskong** [comment]
Fixed #1297.
